accounts/accounts_cli.go

```diff
@@ -0,0 +1,328 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/codegangsta/cli"
+	"github.com/howeyc/gopass"
+)
+
+// Represents the user running the audit on the accounts that they have access to via permissons
+type admin struct {
+	host   string
+	client *http.Client
+	User   struct {
+		Id    string `json:"userid"`
+		Name  string `json:"username"`
+		Token string `json:"-"`
+	} `json:"administrator"`
+	Accounts []*Account `json:"administeredAccounts"`
+}
+
+// Account details that we will audit
+type Account struct {
+	Id      string `json:"userid"`
+	Profile struct {
```

Comment:
in this instance so we could match the "name" to the ptid for jaeb I would leave it